### ISSUE TITLE
WCAG 2.4.7 Focus Visible

### ISSUE THREAD

**Issue by aroubin** [LSCOIT/LegalNavigator#591]
_From @xilin95 on October 31, 2018 3:2_

**Examples where there is no focus:**
- Organizations and videos in Topics & Resources
- FAQ

Make sure each focused item receives a dotted outline or other high-contrast visible highlight to make it obvious where browser focus is.

Focus indicator should be distinguishable by user by any of the following means: Dotted Rectangle (all interactive elements), Background and foreground color change (ex: buttons), Underline with color boldness (ex: links)

**For more info, check: https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html**

_Copied from original issue: Microsoft/Access2Justice#720_

**Comment by KellyeDC-zz** [comment]
Resolved/complete
